stack.yaml

```diff
@@ -9,5 +9,5 @@ packages:
 
 extra-deps:
 - sequence-0.9.8
-- tasty-hedgehog-0.2.0.0
-- tasty-hedgehog-coverage-0.1.0.0
+- tasty-hedgehog-1.0.0.0
+- hedgehog-1.0
```

Comment:
You should simply be able to update the stackage snapshot from `cardano-prelude` to the latest version

Reply:
Then you don't need the hedgehog extra-dep here

Reply:
Yes, here's the link to the latest snapshot

```
resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/a136c4242b9c9f6124b811329bc8ccdfd86c514e/snapshot.yaml


```